### DISCUSSION TITLE
Release schedule - can I help?

### DISCUSSION THREAD

**Discussion by saska** [jitsi-contrib/jitsi-helm#115] [General]
Hey,

I'm doing something where I'd like to have the configurability of the repo in its current state; I was wondering if I could do anything to help speed up the 1.4.0 release.

**Comment by spijet** [comment]
Hello @saska!

I'm a bit hesitant to push a pre-release chart, but you can download the snapshot of this repo as a directory (or a `tar.gz` archive) and use it instead of the public repo version like this:
```
helm -n jitsi --create-namespace install -f jitsi-values.yaml my-jitsi /path/to/jitsi-helm
```

**Reply by saska** [reply]
Yeah, I'm currently running the latest for development purposes but I'm equally hesitant in using a pre-release for production! So I'm just asking whether your roadmap to 1.4.0 has any stuff I could help out with?

Thanks for the quick reply; also, chart's great.

**Reply by spijet** [reply]
Thank you! I'm doing my best (when I have enough time). :)

Basically, for now I'm waiting for updates on #107 to make sure OCTO works properly, and there's an issue in the upstream Jitsi Meet Docker repo that waits for a PR (so that JVB properly configures itself for use with OCTO with WebSockets *and* SCTP).
